docs/source/TROUBLESHOOTING.rst

```diff
@@ -0,0 +1,90 @@
+.. _TROUBLESHOOTING:
+
+Troubleshooting
+===============
+
+We've done our best to make the DeepCell Kiosk robust to common use cases, however, there may be unforseen issues. In the following (as well as on our `FAQ <http://www.deepcell.org/faq>`_, we hope to cover some possible sources of fustration. If you run accross a new problem not listed in either location, please feel free to open an issue on the `DeepCell Kiosk repository <https://www.github.com/vanvalenlab/kiosk>`_.
+
+.. contents:: :local:
+
+Docker Related Errors
+---------------------
+
+Docker Installation
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    DOCKER not defined in docker/build
+    [directory]/kiosk/build-harness/modules/docker/Makefile.build:9: recipe for target 'docker/build' failed
+    make: *** [docker/build] Error 1
+
+**Explanation:** Docker is not installed.
```

Comment:
This is minimally helpful.  Should include a link to the docker installation instructions.